firebase-sessions/src/main/AndroidManifest.xml

```diff
@@ -18,7 +18,14 @@
   <!--<uses-sdk android:minSdkVersion="16"/>-->
 
   <application>
-    <service android:name="com.google.firebase.components.ComponentDiscoveryService"
+    <provider
+      android:name="com.google.firebase.sessions.provider.FirebaseSessionsProvider"
+      android:authorities="${applicationId}.firebasesessionsprovider"
+      android:directBootAware="true"
+      android:exported="false"
+      android:initOrder="100" />
```

Comment:
I assume this is late to initialize. Hopefully that does not create a concern for us? Later we define this, the sessionID generation might get delayed and the underlying SDKs (Fireperf) might be in a wait state to get this sessionID for too long?

Reply:
This will still happen before application oncreate.